include/boost/url/impl/params_encoded_view.ipp

```diff
@@ -119,7 +119,7 @@ find(
 //
 //------------------------------------------------
 
-result<params_encoded_view>
+result<params_view>
```

Comment:
What happened here? We're returning a decoded view instead of encoded? This is an API change

Reply:
Yes. We talked about this before. This would be the only rule to return the encoded result. At the time, you said we could change it later but we had higher priorities. This seemed like a good time to do that because I was already refactoring something that related to this.